mailer/mailer.go

```diff
@@ -68,6 +69,7 @@ func NewMailer(smtp conf.SMTPConfiguration, instanceConfig *conf.Configuration)
 				"price":          price,
 				"hasProductType": hasProductType,
 			},
+			Logger: logrus.New(),
```

Comment:
any chance we could use the actual logger of the http request. it would give us the request id which is very valuable

Reply:
I agree that would be nice. I want to use as light of a touch here as possible. Here's a branch, what do you think abut adding it? https://github.com/netlify/gocommerce/compare/mailer-req-logger

Reply:
I'll put it in a follow-up